client/src/components/container/admin/UsersPage/index.js

```diff
@@ -0,0 +1,80 @@
+import React,{Component} from "react";
+import ReactHTMLTableToExcel from 'react-html-table-to-excel';
+import Axios from "axios";
```

Comment:
The convention is to use lowercase for libraries like this `import axios from 'axios'`, if it's capitalized it indicates that it's a react component